.semaphore/semaphore.yml

```diff
@@ -28,6 +28,16 @@ global_job_config:
       - git fetch --unshallow
 
 blocks:
+  - name: "Build Images"
+    dependencies: []
+    task:
+      jobs:
+        - name: "Build All Images"
+          execution_time_limit:
+            minutes: 120
+          commands:
+            - TEST_IMAGE_BUILD=true make image-all
```

Comment:
I think you should be able to set `TEST_IMAGE_BUILD` in an `env` section in this block somewhere right?

Reply:
Right, yeah. Will do, thanks for pointing that out!